src/main/scala-2.11/org/apache/james/gatling/jmap/JmapMessages.scala

```diff
@@ -0,0 +1,52 @@
+package org.apache.james.gatling.jmap
+
+import java.util.UUID
+
+import io.gatling.core.Predef._
+import io.gatling.http.Predef._
+import org.apache.james.gatling.control.UserFeeder
+
+import scala.util.Random
+
+case class MessageId(id: String)
+case class RecipientAddress(address: String)
+case class Subject(subject: String)
+case class TextBody(text: String)
+
+object JmapMessages {
+
+  def sendMessages(messageId: MessageId, recipientAddress: RecipientAddress, subject: Subject, textBody: TextBody) =
+    JmapAuthentication.authenticatedQuery("sendMessages", "/jmap")
+      .body(StringBody(
+        s"""[[
+          "setMessages",
+          {
+            "create": {
+              "${messageId.id}" : {
+                "from": {"name":"$${username}", "email": "$${username}"},
+                "to":  [{"name":"${recipientAddress.address}", "email": "${recipientAddress.address}"}],
+                "textBody": "${textBody.text}",
+                "subject": "${subject.subject}",
+                "mailboxIds": ["$${outboxMailboxId}"]
+              }
+            }
+          },
+          "#0"
+          ]]"""))
+      .check(status.is(200))
+      .check(jsonPath("$.error").notExists)
+
+  def sendMessagesRandomly(feeder: Array[Map[String, String]]) =
+    sendMessages(
```

Comment:
I improved readability by adding default argument for the case class here.
You just have the argument creation threw constructor, which have the benefit of both being readable and succint.